src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -526,6 +534,23 @@ private Response processDatasetUpdate(String jsonBody, String id, DataverseReque
         }
     }
     
+    private void validateDatasetFieldValues(List<DatasetField> fields) throws JsonParseException {
+        String errors = "";
+
+        for (DatasetField dsf : fields) {
+            if (dsf.getDatasetFieldType().isAllowMultiples() && dsf.getControlledVocabularyValues().isEmpty()
+                    && dsf.getDatasetFieldCompoundValues().isEmpty() && dsf.getDatasetFieldValues().isEmpty()) {
+                errors += "Empty multiple value for field " + dsf.getDatasetFieldType().getDisplayName() + " ";
```

Comment:
should use `StringBuilder` to avoid bad perf in the case of many errors.

Reply:
OK. Done. thanks, Oscar.